docs/messaging/messaging-spans.md

```diff
@@ -211,6 +205,16 @@ The following operations related to messages are defined for these semantic conv
 | `receive`      | A message is received from a destination by a message consumer/server. |
 | `process`      | A message that was previously received from a destination is processed by a message consumer/server. |
 
+### Span kind
+
+[Span kinds](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#spankind)
+SHOULD be set according to the following table, based on the operation a span describes.
+
+| Operation name | Span kind|
+|----------------|-------------|
+| `publish`      | `PRODUCER` |
+| `receive`      | `CONSUMER` |
```

Comment:
we should probably either remove process now or add it to the table here since this section is used to vaguely cover it.

Reply:
In the messaging WG meeting two weeks ago we decided to incorporate OTEP-proposed changes for `create` and `deliver` span into this PR, because otherwise we'll end up with a barely usable intermediate state (e. g. ambiguity about the process operation, or also the fact that we'll only support pull-based scenarios until `deliver` is added too).